org.gimp.BaseApp.json

```diff
@@ -0,0 +1,279 @@
+{
+    "id": "org.gimp.BaseApp",
+    "branch": "stable",
+    "runtime": "org.gnome.Platform",
+    "sdk": "org.gnome.Sdk",
+    "runtime-version": "3.24",
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        },
+    "cleanup": [ "/man", "/share/man", "/share/gtk-doc",
+                 "/share/vala", "*.la", "*.a"]
+    },
+    "modules": [
+        {
+            "name": "webkitgtk",
+            "config-opts": [ "--with-gtk=2.0", "--disable-webkit2", "--disable-geolocation", "--disable-video",
+                             "--disable-spellcheck", "--disable-credential-storage", "--disable-web-audio" ],
+            "build-options" : {
+                "env": {
+                    "PYTHON": "/usr/bin/python3"
+                }
+            },
+            "cleanup": [ "/share/doc", "/bin" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.webkitgtk.org/releases/webkitgtk-2.4.11.tar.xz",
+                    "sha256": "588aea051bfbacced27fdfe0335a957dca839ebe36aa548df39c7bbafdb65bf7"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/webkitgtk-2.4.11.patch"
+                }
+            ]
+        },
+        {
+            "name": "python",
+            "modules": [
+                {
+                    "name": "py2cairo",
+                    "rm-configure": true,
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "http://cairographics.org/releases/py2cairo-1.10.0.tar.bz2",
+                            "sha256": "d30439f06c2ec1a39e27464c6c828b6eface3b22ee17b2de05dc409e429a7431"
+                        },
+                        {
+                            "type": "script",
+                            "commands": [
+                                "libtoolize --force",
+                                "aclocal",
+                                "autoheader",
+                                "automake --force-missing --add-missing --foreign",
+                                "autoconf"
```

Comment:
Pretty sure you can combine these 4 commands into `autoreconf ...`

Reply:
Maybe. I don't remember the history of why I added these and if I tried simply autoreconf. I could make the change but that would be untested either.

Reply:
Ok I made a quick test. It doesn't work. In theory, it would call all these, even libtoolize: https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/autoreconf-Invocation.html
In practice, it failed with:
```
configure.ac:34: installing './compile'
configure.ac:34: installing './config.guess'
configure.ac:34: installing './config.sub'
configure.ac:25: installing './install-sh'
configure.ac:25: installing './missing'
Makefile.am: error: required file './ChangeLog' not found
doc/Makefile.am:32: warning: PAPEROPT_$(PAPER: non-POSIX recursive variable expansion
doc/Makefile.am:55: warning: user target 'clean' defined here ...
automake: ... overrides Automake target 'clean' defined here
doc/Makefile.am:55: consider using clean-local instead of clean
doc/Makefile.am:58: warning: user target 'html' defined here ...
automake: ... overrides Automake target 'html' defined here
doc/Makefile.am:58: consider using html-local instead of html
/usr/share/automake-1.15/am/ltlibrary.am: warning: '_cairo.la': linking libtool libraries using a non-POSIX
/usr/share/automake-1.15/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
src/Makefile.am:11:   while processing Libtool library '_cairo.la'
src/Makefile.am: installing './depcomp'
src/Makefile.am:9: installing './py-compile'
autoreconf: automake failed with exit status: 1
Error: module py2cairo: module py2cairo: Child process exited with code 1
```

The command I replaced these with was:

> "autoreconf --force --install

And I tried a few variants. Reading the doc, it should be ok.
Maybe I am missing something? In any case, I can't really make the time right now to make this script into a oneliner with `autoreconf`, especially since the current manifest works fine. Hopefully you won't consider this a blocker.

Reply:
Its fine.